inclusive-design/idrc
# feat: integrate SCSS compilation into Eleventy build (resolves #47)

## Description

This PR replaces Laravel Mix's SCSS compilation with a built-in node-sass compilation routine addapted from [maxboeck/eleventastic](https://github.com/maxboeck/eleventastic).

## Steps to test

1. Clone branch.
2. Run `npm run start`.
3. Edit any SCSS file or partial.

**Expected behavior:** A reload is triggered with updated CSS.

## Additional information

[eleventy-plugin-sass](https://www.npmjs.com/package/eleventy-plugin-sass) does not yet support a custom output path so I went this route instead.

## Related issues

Resolves #47.
